glean-core/pings.yaml

```diff
@@ -54,3 +54,15 @@ events:
     - https://bugzilla.mozilla.org/show_bug.cgi?id=1512938#c3
   notification_emails:
     - telemetry-client-dev@mozilla.com
+
+deletion_request:
+  description: >
+    This ping is submitted when a user opts out of
```

Comment:
Can you document on why this is needed? I think we need to clearly express our intent here, since this is a ping that will get sent when upload is disabled.

Reply:
I'd like to wait on the phrasing on desktop, then just use that.

Reply:
Ok, but let's file a bug about this if it happens after merging this PR.